evdns.c

```diff
@@ -151,6 +156,13 @@
 
 #define CLASS_INET     EVDNS_CLASS_INET
 
+/* Timeout in seconds for idle TCP connections that server keeps alive. */
+#define SERVER_IDLE_CONN_TIMEOUT 10
+/* Timeout in seconds for idle TCP connections that client keeps alive. */
+#define CLIENT_IDLE_CONN_TIMEOUT 5
+/* Default maximum number of simultaneous TCP client connections that DNS server can hold. */
```

Comment:
Actually I though that you will add settings for all of this defines

Reply:
Sorry, I misunderstood. I'll add.

Reply:
Done.